src/main/java/org/opentripplanner/gtfs/RepairStopTimesForEachTripOperation.java

```diff
@@ -179,30 +162,19 @@ private void filterStopTimes(List<StopTime> stopTimes) {
             int dwellTime = st0.getDepartureTime() - st0.getArrivalTime();
             if (dwellTime < 0) {
                 issueStore.add(new NegativeDwellTime(st0));
-                if (st0.getArrivalTime() > 23 * SECONDS_IN_HOUR
-                        && st0.getDepartureTime() < 1 * SECONDS_IN_HOUR) {
-                    midnightCrossed = true;
-                    st0.setDepartureTime(st0.getDepartureTime() + 24 * SECONDS_IN_HOUR);
-                } else {
-                    st0.setDepartureTime(st0.getArrivalTime());
-                }
+                return false;
             }
-            int runningTime = st1.getArrivalTime() - st0.getDepartureTime();
 
+            int runningTime = st1.getArrivalTime() - st0.getDepartureTime();
             if (runningTime < 0) {
                 issueStore.add(new NegativeHopTime(new StopTime(st0), new StopTime(st1)));
-                // negative hops are usually caused by incorrect coding of midnight crossings
-                midnightCrossed = true;
-                if (st0.getDepartureTime() > 23 * SECONDS_IN_HOUR
-                        && st1.getArrivalTime() < 1 * SECONDS_IN_HOUR) {
-                    st1.setArrivalTime(st1.getArrivalTime() + 24 * SECONDS_IN_HOUR);
-                } else {
-                    st1.setArrivalTime(st0.getDepartureTime());
-                }
+                return false;
```

Comment:
Should we log to issue store here?

Reply:
It is done already on the line above the changes